settings/components/PolicyPropertyDescription/PolicyPropertyDescription.js

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
Please consider importing only required lodash methods instead of importing the whole library.
```suggestion
import { get } from 'lodash';
```

Reply:
Tree shaking in Webpack takes place independently of the way we use our imports: named or not. Still, it is a question, is the tree-shacking going to take place or not. The thing is we are using lodash library, not lodash-es and the former does not decalre any named export, so I assume that using named exports will not give us any performance boost.
Correct me if I'm wrong, please

Reply:
Sounds reasonable about tree shaking so it is just about code style preferences. I see in this repository lodash imports like yours are being used so I assume it would be fine to leave it as is.